exercises/all-your-base/test.sml

```diff
@@ -0,0 +1,55 @@
+(* version 1.1.0 *)
+
+use "all-your-base.sml";
+use "testlib.sml";
+
+infixr |>
+fun x |> f = f x
+
+val testsuite =
+  describe "all-your-base" [
+    test "base_2(1) -> base_10(1)"
```

Comment:
Why aren't you using the descriptions from https://github.com/exercism/problem-specifications/blob/master/exercises/all-your-base/canonical-data.json? I haven't checked every test, but it seems you are not using all the test cases from `canonical-data.json`, is that correct?